src/main/java/com/ericsson/gerrit/plugins/eiffel/listeners/AbstractEventListener.java

```diff
@@ -47,6 +52,17 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractEventListener.class);
 
+    private static final int CORE_POOL_SIZE = 3;
+    private static final int MAXIMUM_POOL_SIZE = 3;
+    private static final long KEEP_ALIVE = 0;
+    private static final TimeUnit TIME_UNIT = TimeUnit.MILLISECONDS;
+    private static final int QUEUE_SIZE = 16384;
```

Comment:
I m not sure about the logic of selecting these values, specially the queue sie of 16384 (may be 16kb)

Reply:
It's just an arbitrary choice for now. The queue size I took from rabbitmq plugin but it's also an arbitrary number as far as I know. It's half the size of a signed short.